flink-tests/src/test/java/org/apache/flink/test/runtime/BlockingShuffleITCase.java

```diff
@@ -95,6 +95,8 @@ private JobGraph createJobGraph(int numRecordsToSend) {
         StreamGraph streamGraph = env.getStreamGraph();
         streamGraph.setGlobalDataExchangeMode(GlobalDataExchangeMode.ALL_EDGES_BLOCKING);
         streamGraph.setScheduleMode(ScheduleMode.LAZY_FROM_SOURCES);
+        // a scheduler supporting batch jobs is required for this job graph. It will be selected
```

Comment:
hmm...you're not really explaining _why_ this is marked as batch job, the other comment has the same issue.